### PR TITLE
Shifts computers to hug walls.

### PR DESCRIPTION
**Pull request by WJohn**
This bothered me.

If any error is seen in the screen or keyboard placement, let me know.

:cl: WJohnston
tweak: Shifted console sprites to now hug against the edge of the tile.
/:cl:

### CODE REVIEW

**Comment by ShizCalev** [comment]
You missed modular computers, different file.

**Comment by Iatots** [comment]
Computers with directional density like 1-sheet windows when?

**Comment by Supermichael777** [comment]
what about freestanding computers like in uploads?

**Comment by ShizCalev** [comment]
Curious why you didn't just override the pixel offsets during init similar to how APCs behave instead of changing the sprites, seems like it would've been a lot less of a headache

https://github.com/tgstation/tgstation/blob/b7a9a2a452466d411115bb9b8d925fd0a02132a3/code/modules/power/apc.dm#L132-L140

Also testing this, disconnecting the monitor with a screwdriver causes them to jump back to their old sprite
image
image

**Comment by WJohn** [comment]
Yep I missed some stuff, I'm working on fixing it.

File: tgstation.dme [hunk 1]
```diff
@@ -1481,7 +1481,6 @@
 #include "code\modules\integrated_electronics\core\integrated_circuit.dm"
 #include "code\modules\integrated_electronics\core\pins.dm"
 #include "code\modules\integrated_electronics\core\printer.dm"
-#include "code\modules\integrated_electronics\core\saved_circuits.dm"
```
Comment (ShizCalev) [inline tgstation.dme]: Intentional?

Comment (WJohn) [reply tgstation.dme]: Uhhh what the hell how'd that happen

**Comment by RemieRichards** [comment]
Bad way of doing it.

**Review by ChangelingRain — CHANGES_REQUESTED** [review]
Adjust the pixel shift instead of changing all the icons, please.

**Comment by WJohn** [comment]
Bleugh, gonna need to do another PR instead. Will get to that later.

**Pull request closed without merging**